### PR TITLE
[14.0] [MIG] sale_product_smart_buttons: Migrated the module in v14.

### PR DESCRIPTION
**Pull request by Chandresh-SerpentCS**
- Module developed by **Pragati Patel**. (Assigned by **Yogesh Mahera**)

### CODE REVIEW

**Comment by Chandresh-SerpentCS** [comment]
@YogeshMahera-SerpentCS Please review the PR. Thanks.

**Comment by JayVora-SerpentCS** [comment]
Pass this to QA @Chandresh-SerpentCS

**Comment by Chandresh-SerpentCS** [comment]
@JayVora-SerpentCS Please assign Ready to test label in this PR. I can not set any label in the PR.

File: sale_product_smart_buttons/LICENSE/LICENSE [hunk 1]
```diff
@@ -0,0 +1,16 @@
+    OpenERP, Open Source Management Solution
+    Copyright (C) 2018-Today Serpent Consulting Services Pvt. Ltd.
```
Comment (ForamSerpentCS) [inline sale_product_smart_buttons/LICENSE/LICENSE]: 2020-Today

File: sale_product_smart_buttons/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        Smart button on Product
+        Smart button for Amount sale on product
+    """,
+    "category": "Products",
+    "license": "AGPL-3",
+    "depends": ["sale_management"],
+    "data": ["views/product_view.xml"],
+    "installable": True,
+    "application": False,
+    "auto_install": False,
```
Comment (ForamSerpentCS) [inline sale_product_smart_buttons/__manifest__.py]: Here, 'auto_install' can be removed as it has False value by default.

File: sale_product_smart_buttons/models/product.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+    def _compute_sales_amt(self):
+        res = {}
+        amt = 0.00
+        sales_amount = 0.0
+        domain = [
+            ("order_id.state", "in", ["sale", "done"]),
+            ("product_id", "in", self.ids),
+        ]
+        order_lines = self.env["sale.order.line"].search(domain)
```
Comment (ForamSerpentCS) [inline sale_product_smart_buttons/models/product.py]: No need to define separate variable for domain as it only used at once.

**Review by ForamSerpentCS — CHANGES_REQUESTED** [review]

**Comment by pragati-patel98** [comment]
@ForamSerpentCS I solved the comments, Can you please review it?

**Review by ForamSerpentCS — APPROVED** [review]

**Pull request merged**